rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -147,6 +147,15 @@
 @ActiveObject
 public class RMCore implements ResourceManager, InitActive, RunActive {
 
+    /** Limits the number of nodes the Resource Manager accepts. >-1 means UNLIMITED, <0 enforces the limit.
```

Comment:
I understand the documentation writes about the total number of nodes. But the implementation assumes the currently added node to be registered inside the allNodes list. That creates confusion. 
Documentation is clarified accordingly and code is clarified as well.